platform/src/main/java/platform/pageobjects/Footer/TermsConditionsFooter.java

```diff
@@ -0,0 +1,182 @@
+package platform.pageobjects.Footer;
+
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import utils.CommonFunctions;
+
+public class TermsConditionsFooter {
+
+	List<String[]> testdata;
+	public WebDriver driver;
+	private static Logger log = LogManager.getLogger(TermsConditionsFooter.class.getName());
+
+	public TermsConditionsFooter(WebDriver driver) {
+		this.driver = driver;
+		PageFactory.initElements(driver, this);
+	}
+
+	@FindBy(xpath = "//a[contains(text(),'https://hiiragi.evidus.com/petz1/group/dfs/dantai/')]")
+	public WebElement casecTerms;
```

Comment:
Plz mention type of element in name itself.
ex: casecTermsLink etc

Reply:
Will change some Web element names

Reply:
Terms and Conditions Pageobject is being handled by Sanae san.

Reply:
Plz coordinate with Sanae san for the same.
Ensure that it is working fine in your code

Reply:
I have updated Terms and Conditions Code.